verification/src/block_verifier.rs

```diff
@@ -32,20 +33,6 @@ pub struct BlockVerifier<P> {
     transactions: TransactionsVerifier<P>,
 }
 
-impl<P: ChainProvider + CellProvider + Clone> ::std::clone::Clone for BlockVerifier<P> {
```

Comment:
I wonder why we impl the trait manually before?

Reply:
https://github.com/rust-lang/rust/issues/26925

Reply:
> opened this issue on 10 Jul 2015

Reply:
In our scenario,  I think it's ok here.